lib/RTCSession.js

```diff
@@ -2632,6 +2652,33 @@ module.exports = class RTCSession extends EventEmitter
       });
   }
 
+  /**
+   * DTMF RTP Sender
+   */
+  _getDTMFRTPSender()
+  {
+    const senders = this._connection.getSenders().find((sender) =>
+    {
+      return sender.track && sender.track.kind === 'audio';
+    });
+
+    if (!senders)
+    {
+      debugerror('No local audio track to send DTMF with');
```

Comment:
We don't use this kind of logs. Please, keep the JsSIP code style. Same below.

Reply:
can you please clarify. Do you mean it should be `sendDTMF() | No local audio track to send DTMF with`

Reply:
`sendDTMF() | no local audio track to send DTMF with`